ios/FluentUI/Controls/ButtonVnext.swift

```diff
@@ -148,6 +148,7 @@ public struct MSFButtonViewButtonStyle: ButtonStyle {
                 }
         }
         .padding(tokens.padding)
+        .frame(maxWidth: .infinity)
```

Comment:
Just FYI:
I'm taking care of the constraint issue on #341 (along with other issues described in the PR).